plugins/catalog-backend/src/processing/DefaultCatalogProcessingOrchestrator.ts

```diff
@@ -361,19 +392,46 @@ export class DefaultCatalogProcessingOrchestrator
 
     for (const processor of this.options.processors) {
       if (processor.postProcessEntity) {
-        try {
-          res = await processor.postProcessEntity(
-            res,
-            context.location,
-            context.collector.forProcessor(processor),
-            context.cache.forProcessor(processor),
-          );
-        } catch (e) {
-          throw new InputError(
-            `Processor ${processor.constructor.name} threw an error while postprocessing`,
-            e,
-          );
-        }
+        let innerRes = res;
```

Comment:
we should have the read and validate steps instrumented too

Reply:
Added these :)